tests/e2e/e2e_test.go

```diff
@@ -196,9 +201,9 @@ done:
 		// all logs are stored under root data dir
 		logsDir = resp.GetClusterInfo().GetRootDataDir()
 
-		for _, v := range resp.ClusterInfo.CustomVms {
+		for _, v := range resp.ClusterInfo.CustomChains {
 			if v.VmId == vmID.String() {
-				blockchainID = v.BlockchainId
+				blockchainID = v.ChainId
```

Comment:
@dasconnor can we avoid using the name `chainId` as equivalent to the AvalancheGo `blockchainID` in the network runner naming conventions? Right now `chainID` typically refers to the EVM ChainID and we usually use blockchainID to refer to the AvalancheGo blockchainID (the hash of the transaction creating the blockchain).